modules/ROOT/pages/users.adoc

```diff
@@ -60,7 +62,25 @@ By default, in every new organization and business group, you get three options:
 Roles and permissions are grouped under organizations (and also, optionally, under business groups). This means that you can only assign roles and permissions that are related to resources that exist in the organization and/or business group that you are selecting.
 --
 
-If necessary, you can also remove user permissions, but keep in mind that if users have been granted their permission through an assigned role, you cannot view or remove those role-based permissions here--you must do it using the xref:roles.adoc[Roles] section.
+If necessary, you can also remove user permissions, but note that if users have been granted their permissions through an assigned role, you can remove role-based permissions using only the xref:roles.adoc[Roles] functionality. You cannot view or remove role-based permissions in the *Users* page.
+
+Because the Roles feature is deprecated, it is best to xref:teams.adoc#opt-in-to-teams[opt in to the Teams feature] to manage permissions in a hierarchy.
+
+To grant permissions to an individual user:
+
+include::partial$include-nav-steps-org-admin.adoc[]
+
+include::partial$include-nav-users.adoc[]
+
+. Select the user whose permissions you want to modify.
+. Select the product that contains the permission you want to add. For more information about the permissions that are available across Anypoint Platform, see xref:permissions-by-product.adoc[Permissions by Product].
+* If you want to grant the user permissions that apply across the current business group:
+.. Select the permissions from the top *Permission(s)* drop-down.
```

Comment:
What does "the top" mean? At the top of the page/window/section...? Also, let's GLOBALLY replace "drop-down" with menu, assuming that our audiences know how to identify and use common UI controls.